litellm/llms/custom_httpx/http_handler.py

```diff
@@ -32,15 +32,19 @@ class AsyncHTTPHandler:
     def __init__(
         self,
         timeout: Optional[Union[float, httpx.Timeout]] = None,
+        event_hooks: Optional[Mapping[str, list[Callable[..., Any]]]] = None,
         concurrent_limit=1000,
     ):
         self.timeout = timeout
         self.client = self.create_client(
-            timeout=timeout, concurrent_limit=concurrent_limit
+            timeout=timeout, concurrent_limit=concurrent_limit, event_hooks=event_hooks
         )
 
     def create_client(
-        self, timeout: Optional[Union[float, httpx.Timeout]], concurrent_limit: int
+        self,
+        timeout: Optional[Union[float, httpx.Timeout]],
+        concurrent_limit: int,
+        event_hooks: Optional[Mapping[str, list[Callable[..., Any]]]] = None,
```

Comment:
same comment here - please add a test for this

Reply:
The test I added for https://github.com/BerriAI/litellm/pull/6290#discussion_r1806681134 indirectly tests this by checking if a `AsyncHTTPHandler` initialization passes the correct parameters to `httpx.AsyncClient`, but let me know if you'd like a separate test.